_automating-configurations/index.md

```diff
@@ -16,7 +16,7 @@ You can automate complex OpenSearch setup and preprocessing tasks by providing t
 In OpenSearch 2.12, configuration automation is limited to ML tasks.
 {: .info}
 
-OpenSearch use case templates provide a compact description of the setup process in a JSON or YAML document. These templates describe automated workflow configurations for conversational chat or query generation, AI connectors, tools, agents, and other components that prepare OpenSearch as a backend for generative models. For template examples, see [Sample templates](https://github.com/opensearch-project/flow-framework/tree/main/sample-templates).
+OpenSearch use case templates provide a compact description of the setup process in a JSON or YAML document. These templates describe automated workflow configurations for conversational chat or query generation, AI connectors, tools, agents, and other components that prepare OpenSearch as a backend for generative models. For custom template examples, see [Sample templates](https://github.com/opensearch-project/flow-framework/tree/main/sample-templates). For OpenSearch-provided templates, see [Workflow templates]({{site.url}}{{site.baseurl}}/automating-configurations/workflow-templates/).
```

Comment:
Should AI be defined on first mention?

Reply:
Not anymore. From [TERMS](https://github.com/opensearch-project/documentation-website/blob/main/TERMS.md):

AI

No need to define as artificial intelligence (AI).